cron/poll_pull_requests.py

```diff
@@ -23,6 +23,16 @@ def poll_pull_requests():
         pr_num = pr["number"]
         __log.info("processing PR #%d", pr_num)
 
+        # If the owner of this PR added their name to the codebase in any way.
+        if gh.prs.owner_name_added_in_diff(api, settings.URN, pr_num, pr['user']['login']):
+            # We will reject the PR with a message about selfishness not being allowed.
+            __log.info("PR %d rejected for selfishness, closing", pr_num)
+            gh.comments.leave_reject_comment_too_selfish(api, settings.URN, pr_num)
+            gh.prs.label_pr(api, settings.URN, pr_num, ["rejected"])
+            gh.prs.close_pr(api, settings.URN, pr)
```

Comment:
You could factor out a reject_with_comment function instead of duplicating this logic.